src/components/InjectedComponents/PageInspector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { PluginUI, PluginConfig } from '../../plugins/plugin'
+
+export interface PageInspectorProps {}
+
+export function PageInspector(props: PageInspectorProps) {
+    return (
+        <>
+            {[...PluginUI.values()].map((x) => (
+                <PluginPageInspectorForEach key={x.identifier} config={x} />
+            ))}
+        </>
+    )
+}
+
+function PluginPageInspectorForEach({ config }: { config: PluginConfig }) {
+    const F = config.pageInspector
+    if (typeof F === 'function') return <F />
```

Comment:
```suggestion
    const Inspector = config.pageInspector
    if (ReactIs.isValidElementType(Inspector)) return <Inspector />
```

see https://npm.im/react-is

Reply:
The typing guardian already constraint this to be a react component.